scripts/getchoco.ps1

```diff
@@ -275,7 +275,9 @@ try {
 
 # Answer yes to all prompts
 choco feature enable --name=allowGlobalConfirmation | Out-Host
```

Comment:
You need to modify `| Out-Host` as described in the Wiki: https://github.com/robe070/cookbooks/wiki/Developer-Checklist
In Particular point (4) for this line of code, but the WHOLE FILE needs to be reviewed in light of this Check List.